enhancements/proposals/1304-external-seeds/README.md

```diff
@@ -0,0 +1,147 @@
+```
+ERROR 2023-08-03 08:35:54,663 [shard 0] init - Bad configuration: invalid value in 'seeds': 'dummy.scylla.svc.cluster.local': std::system_error (error C-Ares:4, dummy.scylla.svc.cluster.local: Not found)
+```
+
+Although the [documentation](https://opensource.docs.scylladb.com/stable/kb/seed-nodes.html) claims, that "**Once the nodes have joined the cluster, seed node has no function**", introducing the option to control the external seeds might cause instability to the ScyllaDB nodes controlled by the operator due to the above.
+To mitigate this, an issue has been opened with ScyllaDB core: [scylladb/scylladb#14945](https://github.com/scylladb/scylladb/issues/14945).
+
+## Design Details
+
+### ScyllaCluster API
+
+The following extension to the existing ScyllaCluster CR API is proposed:
+```go
+// ScyllaClusterSpec defines the desired state of Cluster.
+type ScyllaClusterSpec struct {
+    ...
+	
+	// externalSeeds specifies the external seeds to propagate to ScyllaDB binary on startup as "seeds".
+	// See https://opensource.docs.scylladb.com/stable/kb/seed-nodes.html.
+	// +optional
+	ExternalSeeds []string `json:"externalSeeds,omitempty"`
+}
+```
+
+The provided `externalSeeds` are passed to the sidecar through a new `--external-seeds` flag. StatefulSet reconciliation in ScyllaDB operator is modified accordingly.
+The sidecar's strategy for choosing seeds to provide to ScyllaDB processes is modified to follow the below pseudocode:
+```go
+func chooseSeeds(externalSeeds []string, seedFromLocalDC string) []string {
+  if is first node in DC {
+    if externalSeeds not empty {
+      return externalSeeds
+    }
+    return seedFromLocalDC
+  }
+
+  return append(externalSeeds, seedFromLocalDC)
+}
+```
+The strategy for getting a seed from local DC (`seedFromLocalDC`) is not modified, i.e. for the first running node in the DC, its own address (`broadcast_address`) is selected, otherwise an address of one other running node from its own DC is selected.
+
+It is important that the external seeds, and only the external seeds, are provided as seeds for the first running node in a datacenter, if they are at all provided by the end user. If external seeds were only appended to the node's own IP, it could result in creating split clusters in case of lack of connectivity between the datacenters.
+For the other nodes, a seed from their own DC is appended to the external seeds for increased resiliency.
+
+The above changes allow for configuring multi datacenter ScyllaDB clusters by manually providing external seeds.
+The below examples assume the two datacenters are to be deployed in one Kubernetes cluster.
+
+### Creating a multi datacenter ScyllaDB cluster
+To create a multi datacenter ScyllaDB cluster, an end user would first deploy a ScyllaCluster. For the purpose of this document, let's assume name `simple-cluster`, namespace `scylla1`, datacenter `us-east-1` and a single rack `us-east-1a`.
```

Comment:
nit: use a `-` for better clarity of the namespace name

Reply:
removed the names altogether as they are not used later in the description